mne/channels/tests/test_interpolation.py

```diff
@@ -16,68 +15,95 @@
 event_id, tmin, tmax = 1, -0.2, 0.5
 event_id_2 = 2
 
-raw = io.Raw(raw_fname, add_eeg_ref=False)
-events = read_events(event_name)
-picks = pick_types(raw.info, meg=False, eeg=True, exclude=[])
 
-reject = dict(eeg=80e-6)
-epochs = Epochs(raw, events, event_id, tmin, tmax, picks=picks,
-                preload=True, reject=reject)
+def _load_data():
+    """Helper function to load data."""
+    # It is more memory efficient to load data in a separate
+    # function so it's loaded on-demand
+    raw = io.Raw(raw_fname, add_eeg_ref=False)
+    events = read_events(event_name)
+    picks_eeg = pick_types(raw.info, meg=False, eeg=True, exclude=[])
+    picks_meg = pick_types(raw.info, meg=True, eeg=False, exclude=[])
```

Comment:
is this not enough to test `reset_bads`? https://github.com/mainakjas/mne-python/blob/interpolate_bads_meg/mne/channels/tests/test_interpolation.py#L97

Tests are already a bit slow, so I'd prefer not to add any more `interpolate_bads` calls.